prime-router/src/main/kotlin/Submission.kt

```diff
@@ -46,8 +46,15 @@ class DetailedSubmissionHistory(
     val id: String? = actionResponse?.id
     val destinations = mutableListOf<Destination>()
 
-    val errors = mutableListOf<DetailActionLog>()
-    val warnings = mutableListOf<DetailActionLog>()
+    /**
+     * Error log for the submission.
+     */
+    val errors = mutableListOf<ConsolidatedActionLog>()
```

Comment:
From @jimduff-usds - Change the destination count to only show destinations with itemcount > 0

Reply:
Done